### DISCUSSION TITLE
The future of SHAP

### DISCUSSION THREAD

**Discussion by Siarkmic** [shap/shap#2823] [General]
@slundberg 
Hi. 
I was wondering whether you are planning to support the amazing SHAP package in the foreseeable future ?
Does anyone beside you have the rights to accept/reject the pull requests?

Kind Regards 
Michal

**Comment by grhaonan** [comment]
I think this project is dying as no one works on Issues and PR...

The last merge is back to June 2022

**Reply by Siarkmic** [reply]
That's almost true. 
dsgibbons created the fork that he would like to maintain and develop with the community.
https://github.com/dsgibbons/shap

**Comment by xiaohk** [comment]
You can also use SHAP from Microsoft's [Interpret Library](https://github.com/interpretml/interpret), which is actively maintained.

**Reply by grhaonan** [reply]
this is awsome, appreciate your suggestion

**Reply by Connor56** [reply]
Where can you find the SHAP algorithm in the Interpret Library?

**Reply by xiaohk** [reply]
@Connor56 You can check out [this documentation](https://interpret.ml/docs/shap.html).

**Reply by grhaonan** [reply]
@xiaohk I just read through its doc and it seems like it doesn't support integration with hugging face models or I am on the wrong page? thanks

**Reply by detrin** [reply]
@xiaohk It is true that the project is maintained, but it is not true that the shape part is maintained. See https://github.com/interpretml/interpret/blob/develop/python/interpret-core/setup.py

**Comment by dsgibbons** [comment]
Hold tight. #2943 is coming.

**Comment by dsgibbons** [comment]
Closing, as the project has been revived since this discussion was started.
